ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+# Lapis Manalis
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~#
+# Albion #
+#~~~~~~~~#
+
+# -ii 7A7D
+
+0.0 "--sync--" sync / 00:0839::The Silvan Throne will be sealed off/ window 0,1
+6.0 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+10.6 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+21.6 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+35.6 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+39.2 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+45.3 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+51.7 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+60.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+62.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
```

Comment:
Is 7A7F the damage and 7A7E the self-targeted? If so, -ii 7A7F?

Reply:
Pretty sure `7A7E` is placing the warning indicators on the floor and `7A7F` is the actual boulders dropping.